Data/models/Model.xcdatamodeld/Model13.xcdatamodel/contents

```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
Since this PR is few weeks old, please double check that we did not modify the core data model after you pushed your code. I'm pretty sure we are safe here

Reply:
Yeah the Pr is rebase dand no new model changes have been made since we created this new version. It is brand new model with no relations, it is safe.